packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,262 @@
+| value (html native) | `string`   | No       | undefined | The value of the input field in a checkbox, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) |
+| isDisabled          | `boolean`  | No       | `false`   | Control whether the checkbox is disabled or not.                                                                                                         |
+| isReadOnly          | `boolean`  | No       | `false`   | Control whether the checkbox is readonly or not.                                                                                                         |
+| isInvalid           | `boolean`  | No       | `false`   | Control whether the checkbox is invalid or not.                                                                                                          |
+| isRequired          | `boolean`  | No       | `false`   | Control whether the checkbox is required or not.                                                                                                         |
+
+### Examples:
+
+#### Basic
+
+```tsx
+// Label text
+<Checkbox>is blade awesome?</Checkbox>
+// Helper text
+<Checkbox helperText="Hint: yes">is blade awesome?</Checkbox>
+// uncontrolled
+<Checkbox defaultChecked={true}>is blade awesome?</Checkbox>
+// controlled
+<Checkbox isChecked={true | false} onChange={isChecked => {}}>is blade awesome?</Checkbox>
+```
+
+### Validations
+
+with `isInvalid` & `isRequired` will pass down `required` and `aria-invalid` attributes accordingly
+
+```tsx
+<Checkbox isRequired>is blade awesome?</Checkbox>
+<Checkbox isInvalid={true | false}>is blade awesome?</Checkbox>
+```
+
+
+## `CheckboxGroup` API
+
+
+| Prop          | Type                        | Required | Default     | Description                                                                                                                        |
+| ------------- | --------------------------- | -------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| children      | React.ReactNode             | No       | `undefined` | Accepts multiple Checkboxes                                                                                                        |
+| label         | `string`                    | No       | `undefined` | The label of the group                                                                                                             |
+| labelPosition | `top                        | left`    | No          | `top`                                                                                                                              | The position of the rendered label |
```

Comment:
Need to re-format this as `top | left`, currently the operator is being treated as the ending of the cell.